README.md

```diff
@@ -1,24 +1,314 @@
 # CSS Fundamentals
 
-<iframe width="640" height="480" src="//www.youtube.com/embed/-k-1TU8qq0Q?rel=0&modestbranding=1" frameborder="0" allowfullscreen></iframe><p><a href="https://www.youtube.com/watch?v=-k-1TU8qq0Q">CSS Fundamentals</a></p>
+## Problem Statement
 
+You know the basics of CSS and how to use it to style HTML pages. But you want
+to customize your presentation further. Fortunately, CSS has several fundamental
+powers to explore, including selector types, color values and comments.
```

Comment:
This doesn't feel like a "power" to me....

"In this lesson, we'll build on the basics we've learned.  We'll learn to select elements based on HTML `attributes`, we'll learn to apply colors, and we'll learn how to comment our CSS."